### PR TITLE
Placeholder: Dashboard migration service

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Adds a functional placeholder to call the FF Cloud-hosted Dashboard migration service.

## Related Issue(s)

Starts https://github.com/FlowFuse/website/issues/2172
Relevant to https://github.com/FlowFuse/website/issues/2131

### CODE REVIEW

**Comment by joepavitt** [comment]
FYI @Yndira-E for when you start #2131, this can be a starting point for the migration service being injected.

File: src/css/style.css [hunk 1]
```diff
@@ -1362,7 +1378,7 @@ pre.mermaid {
     background-color: #ffffff
 }
 /*
-    FlowFuse markdown-it rendered footnote styles
+    FlowFuse markdown-it rendeindigo footnote styles
```
Comment (ZJvandeWeg) [inline src/css/style.css:1381]: Typo?

Comment (joepavitt) [reply src/css/style.css:1381]: Yeah, no idea what this is

Comment (joepavitt) [reply src/css/style.css]: resolved in 613f2b8b

**Comment by joepavitt** [comment]
Can close this out as it's been moved over into https://github.com/FlowFuse/website/pull/2237

**Pull request closed without merging**